cardano_node_tests/tests/test_dbsync.py

```diff
@@ -292,3 +292,37 @@ def test_reconnect_dbsync(
         time.sleep(60)
 
         dbsync_utils.check_tx(cluster_obj=cluster, tx_raw_output=tx_output)
+
+    @allure.link(helpers.get_vcs_link())
+    @pytest.mark.order(-10)
+    @pytest.mark.testnets
+    @pytest.mark.smoke
+    def test_epoch(self, cluster: clusterlib.ClusterLib):
+        """Check expected values in the `epoch` table in db-sync."""
+        common.get_test_id(cluster)
+
+        current_epoch = cluster.g_query.get_epoch()
+        epoch = current_epoch - 20 if current_epoch >= 20 else current_epoch
+
+        blocks_data = {"blocks_count": 0, "tx_count": 0}
```

Comment:
Why dictionary and not just two variables?

Reply:
I just prefer to have the info separated by subject (blocks and epoch) than having 4 independent variables.

`blocks_data["blocks_count"] == epoch_data["blocks_count"]
` 

I think is more readable because we imediattly see that we are comparing the same field.

Reply:
OPTIONAL: I don't like dictionaries for usage like this, because values are not type checked, code completion doesn't work for key names, you can make typo in key name and static code checkers will not warn you. I like sensible variable prefix more. Even full `blocks_data_` and `epoch_data_` prefixes would work well in case like this one.

Just mark this as resolved if it is your final decision to keep the dictionaries.

Reply:
Or dataclasses.